docs/AboutUs.md

```diff
@@ -7,53 +7,45 @@ We are a team based in the [School of Computing, National University of Singapor
 
 You can reach us at the email `seer[at]comp.nus.edu.sg`
 
-## Project team
+## CoLAB Project team
 
-### John Doe
+### Fang Samuel
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/samuelfangjw.png" width="200px">
 
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](https://github.com/samuelfangjw)]
+[[portfolio](team/samuelfangjw.md)]
 
-* Role: Project Advisor
-
-### Jane Doe
-
-<img src="images/johndoe.png" width="200px">
-
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
-
-* Role: Team Lead
+* Role: Deliverables and deadlines
 * Responsibilities: UI
 
-### Johnny Doe
+### Vevel Selvam
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/vevek.png" width="200px">
 
-[[github](http://github.com/johndoe)] [[portfolio](team/johndoe.md)]
+[[github](https://github.com/vevek)]
+[[portfolio](team/vevek.md)]
 
-* Role: Developer
-* Responsibilities: Data
+* Role: Integration
+* Responsibilities: Model, Logic
 
-### Jean Doe
+### Nguyen Duc Danh
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/eriksen2411.png" width="200px">
 
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](https://github.com/Eriksen2411)]
+[[portfolio](team/eriksen2411.md)]
 
-* Role: Developer
-* Responsibilities: Dev Ops + Threading
+* Role: Documentation
+* Responsibilities: Storage
 
-### James Doe
+### Li Rouchen
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/lirc572.png" width="200px">
 
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](https://github.com/lirc572)]
+[[portfolio](team/lirc572.md)]
+
+* Role: Code quality
+* Responsibilities: Test
```

Comment:
Perhaps Tests instead of test would be better here.

Reply:
Agreed